1.8/overview/concepts.md

```diff
@@ -1,121 +1,282 @@
+- If not required for managing master node IP changes or as an SSH jumpbox, the bootstrap machine may be shut down after bootstrapping and spun up on demand when new nodes need to be added to the cluster.
+- For more information, see the [system requirements](/docs/1.8/administration/installing/custom/system-requirements/#bootstrap-node).
 
-A Mesos framework is the combination of a Mesos scheduler and an optional custom executor. A framework receives resource offers describing CPU, RAM, etc. from the leading Mesos master, and allocates them for discrete tasks that can be launched on Mesos agent nodes. Mesosphere-certified Mesos frameworks, called DC/OS services, are packaged and available from public [GitHub package repositories][5]. DC/OS services include Mesosphere-certified Mesos frameworks and other applications.
+### <a name="service"></a>Service
 
-# <a name="master"></a>Master
+A DC/OS service is a set of one or more service instances that can be started and stopped as a group and restarted automatically if they exit before being stopped.
 
-A Mesos master aggregates resource offers from all [agent nodes][8] and provides them to registered frameworks. For more details about the Mesos master, read about [Mesos Master Configuration][9].
+- Services is currently just a UI abstraction that translates to Marathon apps and pods in the CLI and API. This distinction will change over time as the name "service" is pushed upstream into component APIs.
+- Sometimes "service" may also refer to a systemd service on the host OS. These are generally considered components and don’t actually run on Marathon or Mesos.
+- A service may be either a system service or a user service. This distinction is new and still evolving as namespacing is transformed into a system-wide first class pattern.
 
-# <a name="mesosdns"></a>Mesos-DNS
+#### <a name="marathon-service"></a>Marathon Service
 
-[Mesos-DNS][10] is a DC/OS component that provides service discovery within the cluster. Mesos-DNS allows applications and services that are running on Mesos to find each other by using the domain name system (DNS), similar to how services discover each other throughout the Internet.
+A Marathon service consists of zero or more containerized service instances. Each service instance consists of one or more containerized Mesos tasks.
 
-# <a name="packagerepository"></a>Package repository
+- Marathon apps and pods are both considered services.
+    - Marathon app instances map 1 to 1 with tasks
+    - Marathon pod instances map 1 to many with tasks
+- Service instances are restarted as a new Mesos Task when they exit prematurely.
+- Service instances may be re-scheduled onto another agent node if they exit prematurely and the agent is down or does not have enough resources any more.
+- Services may be installed directly via the DC/OS API (Marathon) or indirectly via the DC/OS package manager (Cosmos) from a package repository (e.g. [Mesosphere Universe](#mesosphere-universe))
```

Comment:
Uh... The DC/OS package manager (Cosmos) can be accessed via CLI or UI. I didn't specify. The API is different. Should I spell them all out?